config/jobs/tensorflow/minigo/minigo.yaml

```diff
@@ -5,76 +5,61 @@ presubmits:
     branches:
     - master
     decorate: true
+    extra_refs:
+    - org: tensorflow
+      repo: minigo
+      base_ref: master
```

Comment:
Sorry, I wasn't clear. The `extra_refs` are only needed for the periodic job since periodic jobs are triggered based on a cron string and are not inherently associated with a source code ref. The postsubmit and periodic jobs don't need the `extra_refs` field because they are already associated with a specific source code ref that will be checked out automatically by the pod utilities.